### ISSUE TITLE
Different from com.danielcwilson.plugins.googleanalytics?

### ISSUE THREAD

**Issue by hellogerard** [cmackay/google-analytics-plugin#32]
Quick question:

Is there any significant difference between this plugin and https://github.com/danwilson/google-analytics-plugin?  They both look like they do the same thing. Why write another GA plugin when there's already a popular one in use?

Thanks.

**Comment by cmackay** [comment]
When I initially created this plugin, the other plugins I tried did not work and some required downloading of native libraries. I am not entirely familiar with the latest features other plugins provide but many of the plugins are focused on the most common set of analytics features (tracking views, events, and using custom variables). When I built this plugin I intentionally exposed the set and send functions that the native plugins expose, this allows the client app to utilize all of the possible analytics features exposed by the data collection protocol. 

https://developers.google.com/analytics/devguides/collection/protocol/v1/devguide

As far as I am aware none of the plugins expose the set/send functions so although they look similar there are many other analytics calls that can be utilized. Hope that answers your question.

Thanks,

-Craig
